src/Sarif.Converters/FileInfoFactory.cs

```diff
@@ -89,15 +91,17 @@ private void AddFile(PhysicalLocation physicalLocation)
                 filePath = uri.LocalPath;
             }
 
+            FileData fileData = FileData.Create(
```

Comment:
>FileData [](start = 12, length = 8)

We should consider whether `Foo.Create` is sufficiently explicit to allow us to use `var` here.

Reply:
i don't care about having 'var' here. The value of 'var' in my opinion is 1) reduce line column count, 2) allows declarations to line up nicely in multi-var decls.

in this case, we have a single variable and FileData is not an egregiously long type name

---
In reply to: [136626728](https://github.com/Microsoft/sarif-sdk/pull/614#discussion_r136626728) [](ancestors = 136626728)